app/client/src/pages/aboutus.tsx

```diff
@@ -1,52 +1,111 @@
 import * as React from 'react';
-import { Paper, Grid, Typography } from '@material-ui/core';
+import Image from 'next/image';
+import { Grid, Typography, Hidden } from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
 
 const useStyles = makeStyles((theme) => ({
     root: {
         width: '100%',
-        height: '100%',
+        minHeight: '85vh',
```

Comment:
Hmm, what's the issue that's fixed by setting vertical height?

Reply:
Running the page again, I'm not sure what the difference makes so I'll revert to the original setting. I think there may have been an issue with the footer overlapping the content, but I can't remember clearly.